### PR TITLE
using pulseaudio-module-bluetooth

### PR DESCRIPTION
**Pull request by nuthub**
a) I failed to make lukasjapan's version work on my Pi 1 Model B. The sound was jittering and broken. So I decided to use lukasjapan's wonderful agent but pulseaudio instead of SBCAudioSink to decode the a2dp encapsulated audio stream.

b) Furthermore I modified the install script to use spoken words by espeak instead of playing service-login.ogg and service-logout.ogg. (that could be separated from the request above)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,17 +1,19 @@
...
+
 
 ## Installation
 
 Quick Installation for Raspbian:
 
 ```bash
 sudo -i
-bash <(curl -s https://raw.githubusercontent.com/lukasjapan/bt-speaker/master/install.sh)
+bash <(curl -s https://raw.githubusercontent.com/nuthub/bt-speaker/master/install.sh)
```
Comment (jugendhacker) [inline README.md]: To modify this line isn't very useful if should get merged into this repo...

File: README.md [hunk 2]
```diff
@@ -1,17 +1,19 @@
...
 ```
 
-For details refer to the comments in the [install script](https://github.com/lukasjapan/bt-speaker/blob/master/install.sh).
+For details refer to the comments in the [install script](https://github.com/nuthub/bt-speaker/blob/master/install.sh).
```
Comment (jugendhacker) [inline README.md]: Same here...

Comment (jugendhacker) [reply README.md]: I like the idea with the pin and espeak!
And also pulseaudio sounds better, because it's nice sound quality on a Raspberry

Comment (glensc) [reply README.md]: I think these changes got in here because created pull request from their master branch.

**Comment by lukasjapan** [comment]
> a) I failed to make lukasjapan's version work on my Pi 1 Model B. The sound was jittering and broken.

This is interesting, I had lags, bad sound quality and issues with pulse audio.
In fact, that was the main motivation to create bt-speaker.
To bypass pulse entirely and make bluetooth sound work _without_ a sound daemon.

So I think this approach contradicts with the original bt-speaker philosophy and would make more sense in its own repository. It could probably also be achieved by just changing config files without the need of a separate daemon.

I will update the readme to avoid further confusion.


About pin code:
I will look into that, there is an other PR already.


> b) Furthermore I modified the install script to use spoken words by espeak instead of playing service-login.ogg and service-logout.ogg. (that could be separated from the request above)

Starting to add such gimmicks on default would just bloat the dependency tree with features that not everyone will use.
I think the way to go with this one would be to add examples/comments inside the default hook file and let the user decide wether to install and use such features.
Even the current sound playing might be better as an opt-in feature to keep things minimalistic.

**Comment by brknkfr** [comment]
> This is interesting, I had lags, bad sound quality and issues with pulse audio.
> In fact, that was the main motivation to create bt-speaker.
> To bypass pulse entirely and make bluetooth sound work _without_ a sound daemon.
> 
In my case I actually profit from "better sound" with pulseaudio. When you use https://github.com/EHfive/pulseaudio-modules-bt you can profit from the better codecs like LDAC, APTX, APTX-HD and AAC, otherwise you have to use https://github.com/Arkq/bluez-alsa, which is quiet difficult to install and only supports encoding - not decoding - for some of the codecs.

**Pull request closed without merging**